language_service/src/references/tests.rs

```diff
@@ -93,12 +93,12 @@ fn std_callable_ref() {
                     source: "qsharp-library-source:<std>",
                     range: Range {
                         start: Position {
-                            line: 1,
-                            column: 26,
+                            line: 2,
+                            column: 18,
                         },
                         end: Position {
-                            line: 1,
-                            column: 30,
+                            line: 2,
+                            column: 22,
```

Comment:
Why are the lines and columns changing in these tests?

Reply:
Because of my changes in `test_utils.rs` - I think I changed some whitespace in the "Fake stdlib" while moving strings around.